pudb/debugger.py

```diff
@@ -44,12 +44,7 @@
 CONFIG = load_config()
 save_config(CONFIG)
 
-HELP_TEXT = r"""\
-Welcome to PuDB, the Python Urwid debugger.
--------------------------------------------
-
-(This help screen is scrollable. Hit Page Down to see more.)
```

Comment:
How come you consider this line expendable?

Reply:
The first line I didn't really delete because I moved it to the title bar. The "is scrollable" line I consider redundant. It's help text, users automatically know it will be scrollable. I also deleted a redundant "\" in the first line which appears to be an error.

I am a new user and new users just want to pop up the help page quickly to see the key assignments and no other cruft, without having to scroll around. I don't think valuable screen real estate should be wasted.

I personally don't think the license should even be in the help page at all but as a concession to your existing functionality I left it at the bottom on both help pages.

Reply:
I think the scrolling instructions were put there because it was not clear to some people that this was possible. Unfortunately urwid lacks in discoverability sometimes.

Reply:
To be clearer: Could you please add this back?

Reply:
As I said, "PuDB - The Python Urwid Debugger" is in the title now so surely you don't want that? Also, surely you don't want to waste real-estate on that '------' line? So are you only wanting the "is scrollable" line?

Reply:
Does commit 7aeadbe159387b6403ced25d57d41331b71483b6 suffice?

Reply:
Sure, that's fine.